config/example/config.txt

```diff
@@ -396,6 +396,9 @@ STARLIGHT 0
 ## Uncomment to enable auto-stealthing staff who are AFK for more than specified minutes
 # AUTOSTEALTH 30
 
+## Set to 0/1 to disable/enable automatic admin rights for users connecting from the host the server is running on.
+AUTO_LOCAL_ADMIN 0

```

Comment:
Having this disabled by default kinda makes the feature pointless, no?

Reply:
As discussed on Discord I've tweaked things a bit to make it less pointless.